doc/source/whatsnew/v2.2.0.rst

```diff
@@ -604,6 +604,7 @@ Strings
 - Bug in :meth:`Series.str.find` when ``start < 0`` for :class:`ArrowDtype` with ``pyarrow.string`` (:issue:`56411`)
 - Bug in :meth:`Series.str.replace` when ``n < 0`` for :class:`ArrowDtype` with ``pyarrow.string`` (:issue:`56404`)
 - Bug in :meth:`Series.str.startswith` and :meth:`Series.str.endswith` with arguments of type ``tuple[str, ...]`` for ``string[pyarrow]`` (:issue:`54942`)
+- Bug in comparison operations for ``dtype="string[pyarrow_numpy]"`` raising if dtypes can't be compared (:issue:`56008`)
```

Comment:
This is not needed given that's a new dtype in 2.2?

Reply:
no the dtype was new in 2.1

Reply:
Ah, yes, just not the `_numpy` part ;)